Woodoku/tests/Entity/test_woodoku_board.py

```diff
@@ -1,16 +1,15 @@
 import numpy as np
+import pytest
 
-from WoodokuBoard import _WoodokuBoardRepresentation
+from woodoku_board import _WoodokuBoardRepresentation
 
 N = 9
 
 
 class TestWoodokuRepresentation:
-    @classmethod
-    def setup(cls):
-        cls.one_block_lst = [(0, 0)]
-        cls.three_block_lst = [(1, 3), (5, 8), (0, 7)]
-        cls.five_block_lst = cls.three_block_lst + [(2, 1), (0, 8)]
```

Comment:
Why did you remove the set up, can you not call the block list in @pytest.mark.parametrize ?

Reply:
I found that variables associated with the class such as cls.one_block are not in the same scope of @pytest.mark.parametrize

Reply:
I also experiment with @pytest.fixture. It also doesn't work with @pytest.mark.parametrize